### DISCUSSION TITLE
ReadMe Arabic Version

### DISCUSSION THREAD

**Discussion by yosrinajar** [TeamNewPipe/NewPipe#10956] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Hello,
I would like to add an arabic version of the ReadMe doc since there isn't one. May I proceed?

**Comment by AudricV** [comment]
Sure, go ahead.
